primitives/millau/src/lib.rs

```diff
@@ -61,6 +61,20 @@ pub const MAX_UNREWARDED_RELAYER_ENTRIES_AT_INBOUND_LANE: MessageNonce = 1024;
 /// Maximal number of unconfirmed messages at inbound lane.
 pub const MAX_UNCONFIRMED_MESSAGES_AT_INBOUND_LANE: MessageNonce = 1024;
 
+/// Maximal weight of single message delivery transaction on Millau chain.
+///
+/// This value is a result of `pallet_message_lane::Module::receive_messages_proof` weight formula computation
+/// for the case when single message is delivered. The result then must be rounded up to account possible future
+/// runtime upgrades.
+pub const MAX_SINGLE_MESSAGE_DELIVERY_TX_WEIGHT: Weight = 1_000_000_000;
+
+/// Maximal weight of single message delivery confirmation transaction on Millau chain.
+///
+/// This value is a result of `pallet_message_lane::Module::receive_messages_delivery_proof` weight formula computation
+/// for the case when single message is confirmed. The result then must be rounded up to account possible future
+/// runtime upgrades.
+pub const MAX_SINGLE_MESSAGE_DELIVERY_CONFIRMATION_TX_WEIGHT: Weight = 2_000_000_000;
```

Comment:
Actually it is `1_530_101_000` with current weights.

Reply:
Updated actual value (previous has been missing db writes and reads cost): `1_819_511_000`

Reply:
Is there no way for us to get the consts directly from the weight calculation in `receive_messages_proof`?

If not, what's your process for manually updating these values - is it just running the runtime benchmarks? We should make sure it's documented in the code in case others need to update them.

Reply:
You can get actual values just by changing it to `0`. Thanks to [this test](https://github.com/paritytech/parity-bridges-common/pull/598/files#diff-b154f3d20c7411ee6c223dd0e365d823ad8d58afca82a5ae32375e120a4b9ffeR41) you'll get actual values. Then just round them up && put here.

Reply:
Thanks. I still think that we should have that written in the code somewhere though

Reply:
Well, it was just an optimal way for me - probably not the smartest/fastest way. Other may just take weight values and compute result itself, as comment already states. Imo that's enough. Other hints/alternaties on how to do that imo better to leave for some `how-to-plug-in-message-lane-module-into-your-runtime.md` document, which we may create once we'll actually plug it in somewhere,